tests/bots/AccessHelpsTest.php

```
@@ -0,0 +1,157 @@
+    {
+        return \Publish\Helps\encode_value($v, 'decrypt');
+    }
+
+    /** Inject the SQLite PDO into a fresh Database instance via reflection */
+    private function injectPdo(\Publish\MdwikiSql\Database $db): void
+    {
+        $ref = new \ReflectionClass($db);
+        $prop = $ref->getProperty('db');
+        $prop->setAccessible(true);
+        $prop->setValue($db, self::$pdo);
+    }
```

PHPMD (2.15.0)

[warning] Avoid unused private methods such as 'injectPdo'. (undefined)

(UnusedPrivateMethod)